filters/auth/oidc.go

```diff
@@ -327,7 +327,7 @@ func (f *tokenOidcFilter) internalServerError(ctx filters.FilterContext) {
 // https://openid.net/specs/openid-connect-core-1_0.html#CodeFlowSteps
 // 1. Client prepares an Authentication Request containing the desired request parameters.
 // 2. Client sends the request to the Authorization Server.
-func (f *tokenOidcFilter) doOauthRedirect(ctx filters.FilterContext) {
+func (f *tokenOidcFilter) doOauthRedirect(ctx filters.FilterContext, cookies ...*http.Cookie) {
```

Comment:
Why vararg and not a slice?
What is the point of using a slice of pointers instead of values?

Reply:
Thanks for the hint regarding vararg. I replaced it with `[]*http.Cookie`.

The reason for using `[]*http.Cookie` instead of `[]http.Cookie` is the convention/example found in the golang `net/http` package:
* [`func (r *Request) Cookie(name string) (*Cookie, error)`](https://pkg.go.dev/net/http#Request.Cookie)
* [`func (r *Request) Cookies() []*Cookie`](https://pkg.go.dev/net/http#Request.Cookies)

Maybe it makes sense to follow that example to be closer to the `net/http` package? What do you think?